doc/command_ref.rst

```diff
@@ -1759,11 +1759,11 @@ Upgrade-Minimal Command
 | Deprecated aliases: ``update-minimal``
 
 ``dnf [options] upgrade-minimal``
-    Updates each package to the latest available version that provides a bugfix, enhancement
-    or a fix for a security issue (security).
+    Updates each package to the nearest higher available version that provides
```

Comment:
It is much better. I know that it very difficult to find good description of the behavior. I am thinking about skipping `higher` word from the statements. The sentence starts with `Updates` therefore in general it already says `higher`. What do you think?

Reply:
Agree.